books/free-programming-books-bn.md

```diff
@@ -43,6 +44,12 @@
 * [বাংলায় জাভা প্রোগ্রামিং শেখার কোর্স](http://java.howtocode.dev) - Bazlur Rahman, et al. (howtocode.dev)
 
 
+### JavaScript
+
+* [হাতেকলমে জাভাস্ক্রিপ্ট: সম্পূর্ণ বাংলায় হাতেকলমে জাভাস্ক্রিপ্ট শিখুন](https://zonayed.js.org) - Zonayed Ahmed (HTML)
+* [জাভাস্ক্রিপ্ট বাংলা ই-বুক](https://drive.google.com/file/d/17pbdhUaQzD-GxBgR0AQgYChuHiXJc_9D/view?usp=sharing) - MD Abdullah Al Faruk
```

Comment:
Per [the guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#guidelines) we do not accept google drive links. Please remove this book or update to a different link. Once you have done so, please add `(changes made)` to the PR title. Thank you!

Reply:
Ok, I will do it

Reply:
```suggestion
```